### DISCUSSION TITLE
is it possible to ignore unknown flags?

### DISCUSSION THREAD

**Discussion by symonk** [pytest-dev/pytest#8229] [Q&A]
is it possible that given:

```console
pytest --completely-made-up -s -v
```

That the flag is ignored? `parse_known_args()` kind of thing from argparse?

Having a look around the documentation and I cannot spot anything jumping out around it.  is it even possible with a custom plugin to control that globally for the run?

currently:

```
ERROR: usage: __main__.py [options] [file_or_dir] [file_or_dir] [...]
__main__.py: error: unrecognized arguments: --completely-made-up
```

**Comment by nicoddemus** [comment]
Hi @symonk,

I don't _think_ it is possible but I'm not sure; if you take a look at `_pytest.config` searching for "unknown" you might find the relevant code which throws that error and see if the code around it provides a escape hatch (not time to look up myself now sorry).

**Reply by symonk** [reply]
sweet will have a look
